source/process/release-process.md

```diff
@@ -21,197 +21,177 @@ Pre-work for the current release begins at the code complete date of the previou
-    - Backlog is reviewed and tickets that won’t make it are moved to next release
-4. Docs:
-    - Update [Upgrade Guide](http://docs.mattermost.com/administration/upgrade.html#upgrade-guide) for any steps needed to upgrade to new version
+    - Draft Changelog in a WIP PR with updates for highlights, feature additions, known issues, compatibility updates for deprecated features, config.json, [database changes](https://github.com/mattermost/mattermost-server/blob/master/store/sql_upgrade.go#L181), [API changes](https://github.com/mattermost/mattermost-server/commits/master/model/client.go) (search `#api-proposal` and confirm with Dev) and WebSocket event changes; [see example](http://docs.mattermost.com/administration/changelog.html#compatibility)
+    - Update [Upgrade Guide](http://docs.mattermost.com/administration/upgrade.html#upgrade-guide) with any special notes for upgrading to the new version
     - Submit NOTICE.txt PR for any new libraries added from dev, if not added already. The following two files contain a list of dependencies:
         - https://github.com/mattermost/mattermost-webapp/blob/master/package.json
         - https://github.com/mattermost/mattermost-server/blob/master/glide.yaml
-5. **(Team) Major Feature Complete Meeting (10:00am San Francisco time)**:  
-    - Discuss worst bug currently on master
-    - Discuss release highlights for marketing
-    - Review status of last feature PRs to be merged
-6. Dev:
+3. Logistics:
+    - Start posting a daily Zero Bug Balance query (posted until zero bugs or day of release)
+    - Notify community about upcoming release in Reception [see example](https://pre-release.mattermost.com/core/pl/aq434e5dt3ntmfdowhekxjzi4r)
+4. PM:
+    - Review and update [company roadmap](https://about.mattermost.com/direction/) with which major features made it into the release
+    - Review Jira Backlog and move any tickets that will not be merged, to the next release
+5. Dev:
     - Prioritize reviewing, updating, and merging of pull requests for current release until there are no more tickets in the [pull request queue](https://github.com/mattermost/mattermost-server/pulls) marked for the current release
-    - After the cut-off, any PRs that include significant code changes require approval of the release manager before merging.
-7. Marketing:
-    - Prepare bullet points for release announcement
-    - Propose list of key features included in the release GIF and send to marketing lead for review
-    - Draft release headline and send to marketing lead for review
-    - Queue Twitter banners in the Marketing: Graphics channel
+      - After the cut-off, any PRs that include significant code changes, require approval of the release manager before merging
+6. Marketing:
+    - Prepare bullet points and release headline for release announcement and send to Justin to work on
+    - Decide which sections of the release announcement will have an accompanying screenshot / photo
 
```

Comment:
Suggest adding a T-minus 11:

 ### D. (T-minus 11 working days) Major feature testing
1. QA:
    - Prioritize testing merged PRs and resolved tickets
    - Write and update tests in the Release Testing spreadsheet and in Selenium IDE
    - Run Selenium IDE tests for updated areas, note Pass/Fail and date tested in the Release Testing spreadsheet

Reply:
Added to the checklist - PMs can make the final decision.